openfl-tutorials/interactive_api/medmnist_fl_3D_WIP/envoy/medmnist_shard_descriptor.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (C) 2020-2021 Intel Corporation
```

Comment:
If the shard descriptors for 2D and 3D tutorials are the same, let's just keep one folder for director, one for envoy and add different jupyter notebooks for both the tutorials under workspace.

Reply:
the 3D was uploaded at this stage just to look for help to fix a bottleneck. It is not expected to be merged for now. did it appear as a PR to the main branch? Thanks

Reply:
Yes, it is checked in for this PR. It is better to remove `medmnist_fl_3D_WIP` folder completely from this PR.